src/oic/oic/consumer.py

```diff
@@ -461,3 +481,46 @@ def check_session(self):
 
     def end_session(self):
         pass
+
+# LOGOUT related
+
+    def backchannel_logout(self, request=None, request_args=None):
+        """
+        Receives a back channel logout request and returns a Session ID
+        if the request was OK.
+
+        :param request: A urlencoded request
+        :param request_args: The request as a dictionary
+        :return: A Session Identifier
+        """
+        if request:
+            req = BackChannelLogoutRequest().from_urlencoded(as_unicode(request))
```

Comment:
The `as_unicode` is not necessary. `from_urlencoded` can handle both.

Reply:
OK